### DISCUSSION TITLE
add a tilted track options to mania

### DISCUSSION THREAD

**Discussion by The-Grace-God** [ppy/osu#25483] [Ideas]
i know you talked about this in your latest video a slight bit but i think a tilted track would just be a nice skin option in general not just for people who are on mobile but also for people on desktop for the people who would want to use a GH skin or a SDVX skin when playing mania

**Comment by Gabixel** [comment]
So you're proposing something like https://github.com/ppy/osu/discussions/25447 ?

**Reply by The-Grace-God** [reply]
close but as an ini setting instead of an mod
